release/README.md

```diff
@@ -1,70 +1,129 @@
 # Release workflow
 
-This is the recommended release flow in a nutshell for the release 24.7:
+This is the recommended release flow in a nutshell for the release 25.3.
+These steps are repeated for each release tag i.e. for both release-candidates and non-release-candidates:
 
 ```shell
 # start with the product images ...
 # create and push the release branch
-./release/create-release-branch.sh -b 24.7 -w products # Only add the -p flag after testing locally first
+./release/create-release-branch.sh -b 25.3 -w products # Only add the -p flag after testing locally first
 
-# create and push the release tag
-./release/create-release-tag.sh -t 24.7.0 -w products # Only add the -p flag after testing locally first
+# create a new PR dedicated to the current release tag
+./release/create-release-candidate-branch.sh -t 25.3.0-rc1 -w products # Only add the -p flag after testing locally first
+
+# merge the PR branches, once the necessary changes have been made (e.g. cherry-picked from main)
+./release/merge-release-candidate.sh -t 25.3.0-rc1 -w products # Only add the -p flag after testing locally first
+
+# tag the merge commit
+./release/tag-release-candidate.sh -t  25.3.0-rc1 -w products # Only add the -p flag after testing locally first
 
 # monitor the GH action that builds ~80 images for success.
 # build failures alerts will appear in the #notifications-container-images channel.
+# repeat the above with e.g. the tag 25.3.0 once a release candidate is accepted.
```

Comment:
question: Should the `./release/tag-release-candidate.sh` script be used to create a non-rc tag? If so, I think it is a little confusing.

Reply:
This could get philosophical - yes, I see your point, but we have one flow via the testable PR-branches for everything that is pre- or on-the-way-to- release.

Reply:
We could call it tag-release.sh I suppose, but the same flow would/could be used for things that are tagged -rcXXX (the original plan). Not sure if that is much an improvement. Hopefully any ambiguitiy is cleared up in the docs, specifically [here](https://github.com/stackabletech/stackable-utils/blob/feature/release-candidates/release/README.md?plain=1#L89-L93)?

Reply:
Let's leave it as is for now. We still can change this down the line if it annoys us.